lib/mockingbird/httpoison_http_client.ex

```diff
@@ -21,32 +21,50 @@ defmodule Mockingbird.HTTPoisonHttpClient do
           {:ok, HTTPoison.Response.t()} | {:error, HTTPoison.Error.t()}
   def call(verb, url, body \\ %{}, headers \\ %{}, options \\ [])
 
-  def call(:delete, url, _body, headers, options) do
+  def call(verb, url, body, headers, options) do
+    do_call(verb, url, body, headers, options, 0)
+  end
+
+  defp do_call(_verb, _url, _body, _headers, _options, 3) do
```

Comment:
Maybe replace `3` with a constant to make it clear that it represents the max_retries allowed